source/server/admin/admin.h

```diff
@@ -58,6 +58,79 @@ class AdminInternalAddressConfig : public Http::InternalAddressConfig {
   bool isInternalAddress(const Network::Address::Instance&) const override { return false; }
 };
 
+class AdminFactoryContext final : public Configuration::FactoryContext {
+public:
+  AdminFactoryContext(Envoy::Server::Instance& server)
+      : server_(server), scope_(server_.stats().createScope("")),
+        listener_scope_(server_.stats().createScope("listener.admin.")) {}
```

Comment:
This scope creation should be safe because we will create server scope when we creating the server instance. So, the stats scope creation should has been ready when we creating the `AdminFactoryContext`. (IMO)